plugins/woocommerce-blocks/tests/e2e/README.md

```diff
@@ -88,7 +88,7 @@ pnpm run test:e2e:classic-theme
 pnpm run test:e2e:block-theme-with-templates
 ```
 
-_Note: All command parameters of `test:e2e` can be used for these commands too.
+\_Note: All command parameters of `test:e2e` can be used for these commands too.
```

Comment:
@dinhtungdu I thought that too when I first committed it, but its auto markdown formatting... 🤷🏻

Reply:
Oh its because this isn't terminated with another `_` .. so its supposed to be the underscore character. I have no idea what the intent of this was originally.